edx_exams/apps/router/interop.py

```diff
@@ -1,15 +1,24 @@
 """
 Module for writing data to edx-proctoring
 """
+import logging
 from posixpath import join as urljoin
+from urllib.parse import quote_plus
 
 from django.conf import settings
 from requests.exceptions import HTTPError
+from rest_framework import status
+from simplejson import JSONDecodeError
 
 from edx_exams.apps.core.rest_utils import get_client, make_request
 
 LMS_PROCTORING_PLUGIN_BASE_PATH = 'api/edx_proctoring/v1/'
 LMS_REGISTER_PROCTORED_EXAMS_API_TPL = 'proctored_exam/exam_registration/course_id/{}'
+LMS_PROCTORED_EXAM_ACTIVE_ATTEMPT_API_TPL = 'proctored_exam/active_attempt?user_id={}'
```

Comment:
this endpoint to just return the 'current' attempt does not exist but is just a few lines of copy/paste from the existing `proctored_exam/attempt/course_id/{}` to set it up. I'll have a PR for that to follow.